src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -171,6 +171,10 @@ else if (!hasText(leftCurly)) {
                     ast.getText());
             }
         }
+
+        if (ast.getType() == TokenTypes.LITERAL_CASE) {
+            checkForEmptyCase(ast);
+        }
```

Comment:
@ps-sp , "log." statements should be in "visitToken" method only, please think how to make it.